src/StimulusBundle/src/Dto/StimulusAttributes.php

```diff
@@ -152,7 +152,7 @@ public function __toString(): string
         return rtrim(implode(' ', [
             ...$attributes,
             ...array_map(function (string $attribute, string $value): string {
-                return $attribute.'="'.$this->escapeAsHtmlAttr($value).'"';
+                return $attribute.'="'.$this->escape($value, 'html').'"';
```

Comment:
this should probably escape `$attribute` here (and this one needs to use `html_attr`)

Reply:
I thought about it too, I'm adjusting tests and I will see for more improvments

Reply:
Done